VK_App/VK_App/NetworkService/NetworkService.swift

```diff
@@ -55,15 +62,35 @@ final class NetworkService {
             Constants.userID: userId,
             Constants.accessToken: token,
             Constants.extended: Constants.extendedValue,
-            Constants.ownerId: Constants.ownerIDValue,
+            Constants.ownerId: "\(ownerIdValue)",
             Constants.vKey: Constants.vValue,
-            Constants.qKey: Constants.qValue
+            Constants.fieldsKey: Constants.fieldsValue,
+            Constants.qKey: queryValue
         ]
 
         let url = "\(Constants.baseUrl)\(Constants.path)\(request.rawValue)"
-        AF.request(url, parameters: parameters).responseJSON(completionHandler: { response in
-            guard let responses = response.value else { return }
-            print(responses)
-        })
+        AF.request(url, parameters: parameters).responseData { [weak self] response in
+
+            guard let data = response.data else { return }
+            do {
+                let decoder = JSONDecoder()
+                let responses = try decoder.decode(T.self, from: data)
+                completion(responses)
+            } catch {
+                print("error - \(error.localizedDescription)")
```

Comment:
```suggestion
                print("\(error.localizedDescription)")
```
либо убирай, либо в константы